package.json

```diff
@@ -1,8 +1,7 @@
 {
   "name": "@edx/edx-proctoring",
-  "//": "Be sure to update the version number in edx_proctoring/__init__.py",
```

Comment:
any reason why we deleted this?

Reply:
Whoops. That was a mistake. Thanks for catching that.